src/tsc-errors/error-store.ts

```diff
@@ -0,0 +1,57 @@
+import { TscError, TscErrorMatchFlags } from './types';
+
+class ErrorStore {
+  private tscErrors: Map<TscErrorMatchFlags, TscError[]> = new Map();
+
+  public getAllErrors(): TscError[] {
```

Comment:
This method seems to only be used in the tests. If we don't need it in production code, let's remove it